pocket_protector/cli.py

```diff
@@ -428,3 +118,318 @@ def _get_creds(kf,
     _check_creds(kf, creds)
 
     return creds
+
+
+def _check_creds(kf, creds, raise_exc=True):
+    if not kf.check_creds(creds):
+        msg = 'Invalid user email'
+        if creds.name_source:
+            msg += ' (from %s)' % creds.name_source
+        msg += ' or passphrase'
+        if creds.passphrase_source:
+            msg += ' (from %s)' % creds.passphrase_source
+        msg += '. Check credentials and try again.'
+        empty_fields = []
+        if creds.name == '':
+            empty_fields.append('user ID')
+        if creds.passphrase == '':
+            empty_fields.append('passphrase')
+        if empty_fields:
+            msg += ' (Warning: Empty ' + ' and '.join(empty_fields) + '.)'
+
+
+        if raise_exc:
+            raise PPCLIError(msg, 1)
+        return False
+    return True
+
+
+def main(argv=None):
+    cmd = Command(name='pocket_protector', func=None)  # func=None means output help
+
+    # add flags
+    cmd.add('--file', missing='protected.yaml',
```

Comment:
:thumbsup: definitely much better than the previous iteration `if... elif... elif`